packages/queries/src/schema/getTableSchema.ts

```
@@ -0,0 +1,56 @@
+function convertToTypeBoxSchema(jsonSchema: any): TSchema {
+    switch (jsonSchema.type) {
+        case 'string':
+            return Type.String(jsonSchema);
+        case 'number':
+            return Type.Number(jsonSchema);
+        case 'integer':
+            return Type.Integer(jsonSchema);
+        case 'boolean':
+            return Type.Boolean(jsonSchema);
+        case 'object':
+            const properties = {};
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)